tests/unit/app/schemas/test_schemas.py

```diff
@@ -0,0 +1,100 @@
+import pytest
+
+from app.api.api_v1.schemas.document import FamilyDocumentResponse
+from app.api.api_v1.schemas.search import SearchResponseFamilyDocument
+
+CLIMATE_LAWS_DOMAIN_PATHS = [
+    "climate-laws.org",
+    "climate-laws.org/path",
+    "climate-laws.org/path/multiple/elements/",
+    "anything.climate-laws.org/",
+    "anything.climate-laws.org/path/",
+    "anything.climate-laws.org/path/multiple/elemeents",
+]
+NON_CLIMATE_LAWS_DOMAIN_PATHS = [
+    None,
+    "",
+    "not-climate-laws.org/",
+    "sub.not-climate-laws.org",
+    "example.com/path",
+    "not-climate-laws.org/path/multiple/elements/",
+]
+SCHEMES = ["http", "https"]
+
+
+@pytest.mark.parametrize("source_domain_path", CLIMATE_LAWS_DOMAIN_PATHS)
+@pytest.mark.parametrize("scheme", SCHEMES)
+def test_climate_laws_source_url_filtered_from_search(source_domain_path, scheme):
```

Comment:
Do we also want test(s) for browse too - just for completeness :grin:

Reply:
We do need better tests for this filtering, I will raise a ticket